examples/data-objects/inventory-app/src/schema.ts

```diff
@@ -3,24 +3,52 @@
  * Licensed under the MIT License.
  */
 
-import { FieldKinds, SchemaAware, SchemaBuilder, ValueSchema } from "@fluid-experimental/tree2";
+import {
+	AllowedUpdateType,
+	FieldKinds,
+	SchemaAware,
+	SchemaBuilder,
+	ValueSchema,
+} from "@fluid-experimental/tree2";
 
 const builder = new SchemaBuilder("inventory app");
-export const float64 = builder.leaf("number", ValueSchema.Number);
-export const string = builder.leaf("string", ValueSchema.String);
+// REV: I'm not sure why this is necessary...
+const float64 = builder.leaf("number", ValueSchema.Number);
+const string = builder.leaf("string", ValueSchema.String);
 
-export const part = builder.struct("Contoso:Part-1.0.0", {
+const part = builder.struct("Contoso:Part-1.0.0", {
+	// REV: ...e.g. why this isn't just SchemaBuilder.field(FieldKinds.value, ValueSchema.String)
 	name: SchemaBuilder.field(FieldKinds.value, string),
 	quantity: SchemaBuilder.field(FieldKinds.value, float64),
 });
 
-export const inventory = builder.struct("Contoso:Inventory-1.0.0", {
+// REV: Building this up as a series of builder invocations makes it hard to read the schema.
+// Would be nice if instead we could define some single big Serializable or similar that laid the
+// schema out and then pass that in.
+const inventory = builder.struct("Contoso:Inventory-1.0.0", {
 	parts: SchemaBuilder.field(FieldKinds.sequence, part),
 });
 
-export const rootField = SchemaBuilder.field(FieldKinds.value, inventory);
-export type RootField = SchemaAware.TypedField<typeof rootField>;
-
-export const schema = builder.intoDocumentSchema(rootField);
+// REV: The rootField feels extra to me.  Is there a way to omit it?  Something like
+// builder.intoDocumentSchema(inventory)
+const rootField = SchemaBuilder.field(FieldKinds.value, inventory);
+const schema = builder.intoDocumentSchema(rootField);
 
 export type Inventory = SchemaAware.TypedNode<typeof inventory>;
+
+export const schemaPolicy = {
+	schema,
+	initialTree: {
```

Comment:
So this is `SharedTree.context.root` after `schematize` is called?

Reply:
Close - schematize gives an `ISharedTreeView` (it's not on the `SharedTree` directly).

Reply:
I see, so the path from the sharedTree dds (idk if this is a good idea) is `SharedTree.SharedTreeView.context.root`. That seems to be pretty deep for me. As a user, I would just like something like `SharedTree.root` or something like that. I'm assuming there's always going to be some sort of `root` like node

Reply:
Kind of - [`SharedTree.view` is actually deprecated](https://github.com/microsoft/FluidFramework/blob/main/experimental/dds/tree2/src/shared-tree/sharedTree.ts#L57) so I think the only way to get the view is with a `SharedTree.schematize()` call which returns an `ISharedTreeView`.  I don't think there's a direct path from SharedTree to its contents like you're looking for.  I agree that would be nice though, and had a [comment in a similar spirit here](https://github.com/microsoft/FluidFramework/pull/17568/files/d9046e3c3a8b3b0a74f8314f7d3028d6c6076605#diff-f2d1c2970f7f51d80487780aa22bdc305c96bdff393254636f79bcdbf2e8302cR41).